modules/balana-core/src/main/java/org/wso2/balana/xacml3/AllOfSelection.java

```diff
@@ -115,6 +115,9 @@ public MatchResult match(EvaluationCtx context){
 
     }
 
+    public List<TargetMatch> getMatches() {
+        return matches;
```

Comment:
Best practice would be to return the unmodifiable list using 'Collections.unmodifiableList'